### DISCUSSION TITLE
app erstellt und jetzt?

### DISCUSSION THREAD

**Discussion by B-Baumeister** [neuefische/questions#766] [Web: Self-directed learning] [answered]
Hallo zusammen, 

ich würde gerne wissen was ich mit meinen ganzen Dateien tun muss, um diese jetzt nicht in vercel sondern auf einem eigenem Host online stellen zu können. Ich glaube es hat was mit "npm run build" zu tun. Hat jemand vorschläge, was ich mir jetzt anschauen könnte, damit es klappt? 

Lg bonnie

**Comment by gregorsart** [comment, marked as answer]
Ich kann jetzt nur raten, da du nicht genau beschreibst, was das für eine App ist, die du erstellt hast.
Aber ich gehe nun einmal von einer NextJs App aus, die du auf Vercel gehostet hast.

Diese App läuft auf nodeJs Basis. Du kannst sie nicht einfach auf einen gewöhnlichen Webspache hochladen, der z.B. auf Basis von Apache läuft. Du musst eine nodeJs Umgebung haben.

Ich hoffe das hilft ein bisschen,

Gruß newbie
